### PR TITLE
[support] Add link to dev chat #56

### LINKED ISSUE #56: [support] Add link to dev chat
Add link to [dev chat](https://gitter.im/openwisp/development) in the chat section of the [support page](https://openwisp.org/support.html).

We should update the text to mention the development channel and link to it, as well as add a second button "Join the Dev Chat"

### PR DESCRIPTION
**Pull request by yashikajotwani12**
Screenshot from 2021-11-25 22-56-56

Fixes #56

### CODE REVIEW

File: support.html [hunk 1]
```diff
@@ -76,19 +76,29 @@ <h2>Chat</h2>
...
+            <i class="comment icon"></i> Join the support chat
           </a>
         </p>
+         <p class="center buttons">
+          <a href="https://gitter.im/openwisp/development"
+             class="ui big inverted button red" target="_blank">
+            <i class="comment icon"></i> Join the dev chat
+          </a>
+        </p>
+        </div>
```
Comment (nemesisdesign) [inline support.html:101]: please increase indentation inside the new div

File: support.html [hunk 2]
```diff
@@ -76,19 +76,29 @@ <h2>Chat</h2>
...
-            <i class="comment icon"></i> Join the OpenWISP Chat
+            <i class="comment icon"></i> Join the support chat
           </a>
         </p>
+         <p class="center buttons">
+          <a href="https://gitter.im/openwisp/development"
+             class="ui big inverted button red" target="_blank">
+            <i class="comment icon"></i> Join the dev chat
+          </a>
+        </p>
```
Comment (nemesisdesign) [inline support.html:100]: please simply add the new `<a>` in the existing `<p>` element and nothing more (no new div, no flex).

File: support.html [hunk 3]
```diff
@@ -76,19 +76,29 @@ <h2>Chat</h2>
             For quick questions you can reach us
             in the <a href="https://gitter.im/openwisp/general">OpenWISP General</a> chat.
         </p>
+            Developers and contributors coordinate with one another using
+            <a href="https://gitter.im/openwisp/development">OpenWISP Development</a> chat.
```
Comment (nemesisdesign) [inline support.html:80]: this must be in a `<p>` element

File: css/style.css [hunk 4]
```diff
@@ -345,3 +345,8 @@ body.content ul ul{
   height: auto;
   max-width: 100%;
 }
+.Chat_buttons{
+  display: flex;
+  margin: 0 3px 0 3px;
+  justify-content: center;
+}
```
Comment (nemesisdesign) [inline css/style.css:352]: this is not needed except the margin, please simply use `.chat.buttons` and add the additional `chat` class to the `p` element.

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 4 above)
Comment (yashikajotwani12) [reply css/style.css:352]: I will update the changes requested.

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**